components/avatar/avatar.css

```diff
@@ -0,0 +1,54 @@
+@import "../../assets/common-css/sidebar.css";
+@import "../../assets/common-css/navbar.css";
+@import "../../assets/common-css/containers.css";
+@import "../../document-css/doc.css";
+
+.avatar-head, .avatar-type{
+    color: var(--font-color);
+    font-size: 1.7rem;
+}
+.avatar-head{
+    margin: -5px;
+}
+.avatar-type{
+    margin-top: 30px;
+}
+.avatar-content{
+    padding: 20px;
+    font-size: 19px;
+    color: var(--primary-color);
+}
+.avatar-container{
+    display: flex;
+    flex-direction: row;
+    gap: 30px;
+    align-items: center;
+    justify-content: center;
+    padding: 30px;
+    background-color: #dbe0e7;
+    border-radius: 5px;
+    flex-wrap: wrap;
```

Comment:
Suggestion:  _flex-direction: row_ and  _flex-wrap: wrap;_ can be combined into _flex-flow: row wrap_